spring-pulsar-sample-apps/sample-pulsar-binder/src/main/java/org/springframework/pulsar/sample/binder/SpringPulsarBinderSampleApp.java

```diff
@@ -37,21 +37,28 @@ public static void main(String[] args) {
 	}
 
 	@Bean
-	public Supplier<String> timeSupplier() {
-		return () -> String.valueOf(System.currentTimeMillis());
+	public Supplier<Time> timeSupplier() {
+		return () -> new Time(String.valueOf(System.currentTimeMillis()));
 	}
 
 	@Bean
-	public Function<String, EnhancedTime> timeProcessor() {
-		return (rawTime) -> new EnhancedTime(rawTime, "5150");
+	public Function<Time, EnhancedTime> timeProcessor() {
```

Comment:
Before #268 this was not possible because both bindings would attempt to call `setSchema` and things would break. All good now though.